### PR TITLE
The installation playbook should not expect AerisCloud to be in /opt

### LINKED ISSUE #3: make install: expects the install to be under /opt
Namely:

``` bash
> grep -rn opt scripts/
scripts//install/roles/env/tasks/main.yml:19:    line: "source /opt/aeriscloud/scripts/env.sh"
scripts//install/roles/env/tasks/main.yml:42:    line: "source /opt/aeriscloud/scripts/env.sh"
```

These paths should not be hard-coded.

c @MiLk

### PR DESCRIPTION
**Pull request by MiLk**
Fixes #3

### CODE REVIEW

File: scripts/install/roles/env/tasks/main.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 - name: "Install new version (zsh)"
   lineinfile:
     dest: "{{ ansible_env.HOME }}/.zshrc"
-    line: "source /opt/aeriscloud/scripts/env.sh"
+    line: "source {{ install_dir }}/scripts/env.sh"
```
Comment (stelcheck) [inline scripts/install/roles/env/tasks/main.yml:19]: Are you sure this won't break anything else? Should there be a `default: /opt` or something?

**Comment by stelcheck** [comment]
Other than the comment I left, LGTM

File: scripts/install/roles/env/tasks/main.yml (hunk 1 above)
Comment (MiLk) [inline scripts/install/roles/env/tasks/main.yml:19]: The playbook is always run by the install script which specifies the extra var, therefore it should be ok.

Comment (stelcheck) [inline scripts/install/roles/env/tasks/main.yml:19]: OK, LGTM then

**Pull request merged**